Source/AutoExtern.Test/Tests.cs

```diff
@@ -24,9 +24,12 @@ public class IntegrationTests {
     startInfo.ArgumentList.Add("NS"); // Namespace
     startInfo.ArgumentList.Add("Library.dfy.template"); // Template
     startInfo.ArgumentList.Add("CSharpModel.dfy"); // Target for CSharpModel.dfy
+    startInfo.ArgumentList.Add("--skip-interface");
+    startInfo.ArgumentList.Add("NS.ICanClone");
     startInfo.ArgumentList.Add("Library.dfy"); // Target file
     startInfo.ArgumentList.Add("Library1.cs"); // Source file
     startInfo.ArgumentList.Add("Library2.cs"); // Source file
+    startInfo.ArgumentList.Add("Library3.cs"); // Source file
```

Comment:
Where is this Libary3.cs coming from?

Reply:
Oh, interesting. I didn't write this code myself, so I'm not sure, but I notice an earlier commit that renamed `Library1.cs`, at least. This test may need to be updated. I'll investigate.